<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>410</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>840</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2520</width>
    <height>1227</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Агаш Пазл - Справочники</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>841</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,5,0,0,0">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>50</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>50</number>
    </property>
    <property name="bottomMargin">
     <number>30</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel_2">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Справочники от &quot;Агаш Пазл&quot;

Выберите интересующий справочник ниже</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnWoodDirectory">
      <property name="text">
       <string>Виды древесины</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnThicknessDirectory">
      <property name="text">
       <string>Толщина фанеры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnPlywoodCards">
      <property name="text">
       <string>Фанерные листы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
